docs/docs/dev/index.md

```diff
@@ -17,9 +17,16 @@ The instructions here are for macOS; where appropriate, they'll proffer a Linux
 
 ### Install your dependencies
 
-On macOS, that means installing copier, uv, just, and gsed. For simplicity these
-instructions will assume that you have homebrew installed, and follow the
-instructions to set your PATH for each tool.
+On macOS, that means installing the following:
+
+* Just
+* gsed
+* Python 3.12 (ideally using mise)
```

Comment:
I'm going to remove this; `uv` will install this if needed, so there's no need to pre-install it.

```suggestion
```